modules/caddyhttp/reverseproxy/metrics.go

```diff
@@ -0,0 +1,80 @@
+package reverseproxy
+
+import (
+	"runtime/debug"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"go.uber.org/zap"
+)
+
+var reverseProxyMetrics = struct {
+	init             sync.Once
+	upstreamsHealthy *prometheus.GaugeVec
+}{
+	init: sync.Once{},
+}
+
+func initReverseProxyMetrics() {
+	const ns, sub = "caddy", "reverseproxy"
```

Comment:
```suggestion
	const ns, sub = "caddy", "reverse_proxy"
```

Technically, the module name is `http.handlers.reverse_proxy` -- not sure how much we care to have namespace parity here?

Reply:
This makes sense. Internally it's `reverseproxy`, but externally it's always `reverse_proxy`. I'll change this to `reverse_proxy` to avoid confusing users. Better to stay with consistent naming.